sdk/python/feast/infra/offline_stores/remote.py

```diff
@@ -38,38 +46,19 @@ class RemoteRetrievalJob(RetrievalJob):
     def __init__(
         self,
         client: fl.FlightClient,
-        feature_view_names: List[str],
-        name_aliases: List[Optional[str]],
-        feature_refs: List[str],
-        entity_df: Union[pd.DataFrame, str],
-        project: str,
-        full_feature_names: bool = False,
+        api: str,
+        api_parameters: Dict[str, Any],
```

Comment:
how about changing these parameters. names `api -> command` and `api_parameters -> request parameters` 
 this will change other place as well and within server  `command['api'] -> command['command']` 
 WDYT? 
cc @dmartinol

Reply:
I think I prefer the current names

Reply:
let's try to be consistent, community reviewers can propose their own names